packages/loader/container-loader/src/containerStorageAdapter.ts

```diff
@@ -126,12 +128,25 @@ export class ContainerStorageAdapter implements IDocumentStorageService, IDispos
 	}
 
 	public async getSnapshot(snapshotFetchOptions?: ISnapshotFetchOptions): Promise<ISnapshot> {
-		if (this._storageService.getSnapshot !== undefined) {
-			return this._storageService.getSnapshot(snapshotFetchOptions);
+		if (this._storageService.getSnapshot === undefined) {
+			throw new UsageError(
+				"getSnapshot api should exist in internal storage in ContainerStorageAdapter",
+			);
+		}
+
+		const key = snapshotFetchOptions?.loadingGroupIds?.sort().join();
```

Comment:
Is it key, or keys? loadingGroupIds suggests plural.

Reply:
Converted to just key, it should just work for any sort of groupId calls.